test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -314,7 +314,7 @@ public class HubEndpointTests
             var hubProtocolMock = new Mock<IHubProtocol>();
             hubProtocolMock.Setup(m => m.Name).Returns("CustomProtocol");
 
-            dynamic endPoint = HubEndPointTestUtils.GetHubEndpoint(typeof(HubT));
+            EndPoint endPoint = HubEndPointTestUtils.GetHubEndpoint(typeof(HubT));
```

Comment:
Can these be var now?

Reply:
Also the variable results from ConnectAsync can be var instead of task.

Reply:
Done!